### ISSUE TITLE
Pressing delete when typing text in wysiwyg editor removes the selected block

### ISSUE THREAD

**Issue by code-guru** [wbkd/react-flow#560]
When user types inside wysiwg editor and press the delete button on keyboard, react-flow would delete the selected node instead of expected behaviour of only deleting the last typed character. In the code [here](https://github.com/wbkd/react-flow/blob/main/src/utils/index.ts#L6), there is a check to make sure the event is not coming from an input field , however `div` with `contenteditable` attributes also behaves like an input field.

It looks there to add new check for that use, maybe something like: 
```
if(e && e.target.attributes['contenteditable']) {
return true;
}
```

**Comment by moklick** [comment]
good point :) we will add that.

**Comment by moklick** [comment]
released in 6.1.2
